src/App.tsx

```diff
@@ -462,6 +463,9 @@ function AppContent() {
                 <Route exact path={APP_PATHS.PROJECT_VIEW}>
                   <ProjectView />
                 </Route>
+                <Route exact path={APP_PATHS.PROJECT_EDIT}>
+                  <ProjectEditView />
+                </Route>
               </>
```

Comment:
I found that placing multiple `Route` components within a fragment like this causes problems. The `Switch` component doesn't like any direct children other than `Route`s or `Redirect`s. So you'll need to expand this out and individually wrap each `Route` in a conditional rendering block, like
```typescript
{featureFlagProjects && (
  <Route exact path={APP_PATHS.PROJECTS}>
    {getProjectsView()}
  </Route>
)}
{featureFlagProjects && (
  <Route exact path={APP_PATHS.PROJECTS_NEW}>
    <NewProjectFlow reloadData={reloadProjects} />
  </Route>
)}
{featureFlagProjects && (
  <Route exact path={APP_PATHS.PROJECT_VIEW}>
    <ProjectView />
  </Route>
)}
{featureFlagProjects && (
  <Route exact path={APP_PATHS.PROJECT_EDIT}>
    <ProjectEditView />
  </Route>
)}
```
Which is uglier, but I believe necessary

Reply:
Ended up splitting into its own router